shared/chat/conversation/messages/popup.native.js

```diff
@@ -140,9 +141,18 @@ type OwnProps = MessagePopupRouteProps & {}
 
 export default connect(
   (state: TypedState, {routeProps}: OwnProps) => {
+    // $FlowIssue doesn't like routeProps here
```

Comment:
I wish there were an easy way to merge this shared code with the desktop version. This is fine for now, though

Reply:
same! I'll see how it looks after Nojima's PR goes in